docker/scripts/ci_start.sh

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2020 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+APOLLO_ROOT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+source "${APOLLO_ROOT_DIR}/scripts/apollo.bashrc"
+
+CACHE_ROOT_DIR="${APOLLO_ROOT_DIR}/.cache"
+
+DOCKER_REPO="apolloauto/apollo"
+DEV_INSIDE="in-dev-docker"
+
+HOST_ARCH="$(uname -m)"
+TARGET_ARCH="$(uname -m)"
+USER_VERSION_OPT=
+
+DOCKER_RUN="docker run"
+FAST_MODE="no"
+USE_LOCAL_IMAGE="no"
+USE_GPU_HOST=0
+
+VOLUME_VERSION="latest"
+USER_SPECIFIED_MAP=
+MAP_VOLUME_CONF=
+OTHER_VOLUME_CONF=
+
+DEFAULT_MAPS=(
+  sunnyvale_big_loop
+  sunnyvale_loop
+  sunnyvale_with_two_offices
+  san_mateo
+)
+
+DEFAULT_TEST_MAPS=(
+  sunnyvale_big_loop
+  sunnyvale_loop
+)
+
+eval $(grep ^VERSION_X86_64= ${APOLLO_ROOT_DIR}/docker/scripts/dev_start.sh)
+eval $(grep ^VERSION_AARCH64= ${APOLLO_ROOT_DIR}/docker/scripts/dev_start.sh)
+
+function parse_arguments() {
+    local custom_version=""
+
+    while [ $# -gt 0 ] ; do
+        local opt="$1"; shift
+        case "${opt}" in
+
+        -f|--fast)
+            FAST_MODE="yes"
+            ;;
+
+        -l|--local)
+            USE_LOCAL_IMAGE="yes"
+            ;;
+
+        *)
+            warning "Unknown option: ${opt}"
+            exit 2
+            ;;
+        esac
+    done # End while loop
+
+    [ -n "${custom_version}" ] && USER_VERSION_OPT="${custom_version}"
```

Comment:
If you don't want to support `-t` option, please remove `custom_version` related logic.